crates/ruff_python_ast/src/visitor/source_order.rs

```diff
@@ -550,6 +561,23 @@ where
     visitor.leave_node(node);
 }
 
+#[inline]
+pub fn walk_f_string_value<'a, V>(visitor: &mut V, f_string: &'a FStringValue)
+where
+    V: SourceOrderVisitor<'a> + ?Sized,
+{
+    for f_string_part in f_string {
+        match f_string_part {
+            FStringPart::Literal(string_literal) => {
+                visitor.visit_string_literal(string_literal);
+            }
+            FStringPart::FString(f_string) => {
+                visitor.visit_f_string(f_string);
+            }
+        }
+    }
+}
```

Comment:
I'd prefer not to make changes to the source order visitor as part of this PR because changes to the visitor may break existing code (the code here should call `enter_node` and `exit_node` for every visited node and visitors that override those methods may now behave differently because they see additional nodes).